// ng2-bootstrap-grid/dist/out-tsc/index.js
"use strict";
// import { BootstrapGridModule } from './src/bootstrap-grid.module';
// import { BootstrapGridModule } from './index';
// import {NgModule, ModuleWithProviders} from '@angular/core';
// import {CommonModule} from '@angular/common';
Object.defineProperty(exports, "__esModule", { value: true });
// import { BootstrapGridComponent } from './src/bootstrap-grid/bootstrap-grid.component';
// import { ContainerComponent } from './src/container/container.component';
// import { RowComponent } from './src/row/row.component';
// import { ColSmDirective, ColMdDirective, ColLgDirective } from './src/columns.directive';
// import { ColumnComponent } from './src/column/column.component';
// import { AlignCenterDirective, AlignLeftDirective, AlignRightDirective, AlignCenterLeftDirective, AlignCenterRightDirective } from './src/directives/alignments.directive';
// export * from './src/bootstrap-grid/bootstrap-grid.component';
// export * from './src/container/container.component';
// export * from './src/row/row.component';
// export * from './src/columns.directive';
// export * from './src/column/column.component';
// export * from './src/directives/alignments.directive';
var bootstrap_grid_component_1 = require("./src/bootstrap-grid/bootstrap-grid.component");
exports.BootstrapGridComponent = bootstrap_grid_component_1.BootstrapGridComponent;
var container_component_1 = require("./src/container/container.component");
exports.ContainerComponent = container_component_1.ContainerComponent;
var row_component_1 = require("./src/row/row.component");
exports.RowComponent = row_component_1.RowComponent;
var columns_directive_1 = require("./src/columns.directive");
exports.ColSmDirective = columns_directive_1.ColSmDirective;
exports.ColMdDirective = columns_directive_1.ColMdDirective;
exports.ColLgDirective = columns_directive_1.ColLgDirective;
var column_component_1 = require("./src/column/column.component");
exports.ColumnComponent = column_component_1.ColumnComponent;
var alignments_directive_1 = require("./src/directives/alignments.directive");
exports.AlignCenterDirective = alignments_directive_1.AlignCenterDirective;
exports.AlignLeftDirective = alignments_directive_1.AlignLeftDirective;
exports.AlignRightDirective = alignments_directive_1.AlignRightDirective;
exports.AlignCenterLeftDirective = alignments_directive_1.AlignCenterLeftDirective;
exports.AlignCenterRightDirective = alignments_directive_1.AlignCenterRightDirective;
var bootstrap_grid_module_1 = require("./src/bootstrap-grid.module");
exports.BootstrapGridModule = bootstrap_grid_module_1.BootstrapGridModule;
// @NgModule({
//   imports: [
//     CommonModule
//   ],
//   declarations: [
//     BootstrapGridComponent, 
//     ContainerComponent, 
//     RowComponent,
//     ColumnComponent,
//     AlignCenterDirective, AlignLeftDirective, AlignRightDirective, AlignCenterRightDirective, AlignCenterLeftDirective
//   ],
//   exports: [
//     BootstrapGridComponent, 
//     ContainerComponent, 
//     RowComponent,
//     ColumnComponent,
//     AlignCenterDirective, AlignLeftDirective, AlignRightDirective, AlignCenterRightDirective, AlignCenterLeftDirective
//   ]
// })
// export class BootstrapGridModule {
//   // static forRoot(): ModuleWithProviders {
//   //   return {
//   //     ngModule: BootstrapGridModule
//   //   };
//   // }
// }
